mrt/meetings/participant.py

```diff
@@ -580,28 +583,28 @@ class ParticipantsExport(PermissionRequiredMixin, MethodView):
                            'manage_participant')
 
     def get(self):
-
         participants = get_participants_full(g.meeting.id)
 
-        columns = []
-
         custom_fields = (
             g.meeting.custom_fields
-            .filter_by(custom_field_type=CustomField.PARTICIPANT,
-                       is_primary=True)
+            .filter_by(custom_field_type=CustomField.PARTICIPANT)
             .order_by(CustomField.sort))
-        cfs = [cf.slug for cf in custom_fields
-               if cf.field_type.code not in ('image',)]
-        form = ParticipantDummyForm()
-        header = [str(form._fields[k].label.text) for k in columns]
-        header.extend([cf.title() for cf in cfs])
-        columns += cfs
+
+        columns = [cf.slug for cf in custom_fields
+               if cf.field_type.code not in ('image', 'document')]
```

Comment:
Avoid using hard-coded values, when possible:

    if cf.field_type not in (CustomField.IMAGE, CustomField.DOCUMENT)